web/src/app/main.component.ts

```diff
@@ -60,11 +61,27 @@ export class MainComponent {
     ];
 
     constructor(
-        public authService: AuthenticationService,
+        private authService: AuthenticationService,
```

Comment:
Question here, why is the object being passed to the constructor private? Is this some strange javascript voodoo :D

Reply:
Yep 😂. In typescript and angular if `private` in the ts component then it won't be available to the corresponding `html`